dashboard/model_layer.py

```diff
@@ -158,7 +158,9 @@ def enrich_session_info(se):
     se.data_transferred = se.client_bytes_sent + se.client_bytes_received
     se.started = humanize.naturaltime((datetime.utcnow() - se.created_at).total_seconds())
     se.status = 'Ongoing' if ((se.node_updated_at or se.client_updated_at) >= datetime.utcnow() - timedelta(minutes=NODE_AVAILABILITY_TIMEOUT)) else 'Completed'
-
+    se.shortened_node_key = (se.node_key[:6] + '..' + se.node_key[-4:]) \
+        if se.node_key and len(se.node_key) == models.IDENTITY_LENGTH_LIMIT \
+        else se.node_key
```

Comment:
3 lines of inline if statements looks very unreadable - a simple `if` statment should look much better:+    
```py
if se.node_key and len(se.node_key) == models.IDENTITY_LENGTH_LIMIT:
    key = (se.node_key[:6] + '..' + se.node_key[-4:])
else:
    key = se.node_key
se.shortened_node_key = key
```